doc/user/content/sql/functions/normalize.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "normalize function"
+description: "Returns a string normalized to the specified Unicode normalization form."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`normalize` converts a string to a specified Unicode normalization form.
+
+## Signatures
+
+```mzsql
+normalize(str)
+normalize(str, form)
```

Comment:
Oh ... we're just starting to slowly (and I mean slowly) move towards taking the 1st step towards being able to test our docs examples by having the syntax and examples be in a yaml file.
See https://github.com/MaterializeInc/materialize/blob/main/doc/user/data/examples/alter_sink.yml

Then to include in the nomalize.md file, we would just do
{{% include-example file="examples/alter_sink"
example="syntax" %}}
... (the .yml is excluded from the file name, and example="<name>")

This way, even before we get the testing all set up, at least when people change the syntax, it improves the chances that they'll update the examples too.

If amenable, could you try incorporating into the new format?